vnext/Shared/Shared.vcxitems.filters

```diff
@@ -341,6 +341,7 @@
     <ClInclude Include="$(MSBuildThisFileDirectory)Threading\MessageQueueThreadFactory.h">
       <Filter>Header Files\Threading</Filter>
     </ClInclude>
+    <ClInclude Include="$(MSBuildThisFileDirectory)Utils\CppWinrtLessExceptions.h" />
```

Comment:
is this part of the change needed?

Reply:
Hmm I'm not sure where this came from. I'll remove it and make sure the project still builds.